src/main/java/seedu/trippie/command/SearchCommand.java

```diff
@@ -32,12 +32,10 @@ public boolean isExit() {
     public void execute(Ui ui, TrippieData trippieData) {
         List<Place> list = trippieData.getCurrentTrip().getPlaceListObject().getPlaceList();
         int size = list.size();
-        String listPlurality = size > 1 ? "are" : "is";
-        String placePlurality = size > 1 ? "places" : "place";
         if (size == 0) {
             System.out.println("There is currently nothing in your place list. Why not add one?");
         } else {
-            System.out.printf("Here %s the matching %s in your list:%n", listPlurality, placePlurality);
+            System.out.printf("Here is your search result:%n");
```

Comment:
The search result may be more than 1, the previous method to determine plurality can handle this.